pyproject.toml

```diff
@@ -87,3 +81,93 @@ task-tags = ["Example", "todo", "TODO", "FIXME"]
 
 [tool.codespell]
 ignore-words-list = 'edn,juxt'
+
+[build-system]
+requires = ["setuptools>=65", "setuptools_scm[toml]>=7.1"]
+
+[tool.setuptools]
+packages = ["rocky", "octopoes", "keiko", "boefjes", "mula", "bytes"]
+
+[tool.setuptools_scm]
+write_to = "_version.py"
+
+[project]
+name = "openkat"
+version = "0.0.1.dev1"
```

Comment:
We shouldn't specify version because we use setuptools-scm, instead `dynamic = ["version"]` should be added under `[project]`, see also the documentation on https://github.com/pypa/setuptools_scm.